frontend/pages/community.tsx

```diff
@@ -0,0 +1,120 @@
+import { BaseFont, GlobalStyle, H3 } from '../components/common'
+
+const MainContainer = styled(BaseFont)`
+  text-align: center;
+`
+
+const SponsorContainer = styled.div`
+  display: block;
+  max-width: 600px;
+  margin: 0 auto;
+  text-align: left;
+  padding-top: 20px;
+`
+
+export default function SponsorsPage(): JSX.Element {
+  return (
+    <MainContainer>
+      <GlobalStyle />
+      <Meta />
+      <Header />
+      <H3>Community</H3>
+
+      <SponsorContainer>
+        Shields.io is possible thanks to the people and companies who donate
+        money, services or time to keep the project running.
+      </SponsorContainer>
+
+      <SponsorContainer>
+        <h4>Sponsors</h4>
+        ❤️ These companies help us by donating their services to shields:
+        <ul style={{ listStyleType: 'none' }}>
+          <li>
+            <a href="https://www.heroku.com/">
+              <img
+                alt="heroku logo"
+                height="80"
+                src="/images/heroku-logotype-horizontal-purple.svg"
+              />
+            </a>
+          </li>
+          <li>
+            <a href="https://nodeping.com/">
+              <img alt="nodeping logo" height="80" src="/images/nodeping.svg" />
+            </a>
+          </li>
+          <li>
+            <a href="https://sentry.io/">
+              <img
+                alt="sentry logo"
+                height="80"
+                src="/images/sentry-logo-black.svg"
+              />
+            </a>
+          </li>
+        </ul>
+        💵 These organisations help keep shields running by donating on
+        OpenCollective. Your organisation can support this project by{' '}
+        <a href="https://opencollective.com/shields#sponsor">
+          becoming a sponsor
+        </a>
+        . Your logo will show up here with a link to your website.
+        <p>
+          <object data="https://opencollective.com/shields/sponsors.svg?avatarHeight=80&width=600" />
+        </p>
+      </SponsorContainer>
+
+      <SponsorContainer>
+        <h4>Backers</h4>
+        💵 Thank you to all our backers who help keep shields running by
+        donating on OpenCollective. You can support this project by{' '}
+        <a href="https://opencollective.com/shields#backer">
+          becoming a backer
+        </a>
+        .
+        <p>
+          <object data="https://opencollective.com/shields/backers.svg?width=600" />
+        </p>
+      </SponsorContainer>
+
+      <SponsorContainer>
+        <h4>Contributors</h4>
+        🙏 This project exists thanks to all the nice people who contribute
+        their time to work on the project.
+        <p>
+          <object data="https://opencollective.com/shields/contributors.svg?width=600" />
+        </p>
+      </SponsorContainer>
+
+      <SponsorContainer>
+        ✨ Shields is helped by these companies which provide their products or
+        services to open source projects for free:
```

Comment:
Would LGTM and Uptime Robot fit in this section as well?

Reply:
Yeah this is a good point. I added the stuff listed in https://github.com/badges/shields/issues/3737 but there are probably a bunch of other things where we're using some kind of free plan.. netlify fits into this category as well. Can you think of anything else?

LGTM is already listed as a sponsor, but we do use their service too.

Reply:
I've added the ones we've thought of. If more come up, we can always add them later..